.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
 max-line-length = 88
-ignore = E501, E203, W503, ANN101, ANN102
+ignore = E501, E203, W503, ANN101, ANN102, SIM106
```

Comment:
I see that we ignore `SIM106` in `poetry` as well. So I'm fine here. In many cases this warning helps to improve the code. But I've no idea how often it is not helpful in our cases. Maybe we should evaluate it some point later, and only ignore it case-by-case then.